### DISCUSSION TITLE
Hey, I'm possibly interested in being a developer!

### DISCUSSION THREAD

**Discussion by kwaspek104** [triplea-game/triplea#10912] [New Contributor]
Hi, my name's Kasper and I'm interested in possibly being a developer! Motivation is mostly that there are some bugs/wrong rules in certain versions of the game and I want to fix them! Thanks for any help! 

I was told to tag @RoiEXLab, @DanVanAtta & @asvitkine  !

**Comment by beelee1** [comment]
:+1:

**Comment by DanVanAtta** [comment]
I'm going to be mostly afk for the next two weeks, so responses may be limited from me. @RoiEXLab  can be a great resource too - though between the lot of us, should be able to field any questions. Do you have any first bugs in mind? Are you up and running in a dev environment without issue? Let us know how we can help

**Comment by RoiEXLab** [comment]
Hi Kasper,

great to hear that. As @DanVanAtta already hinted the first step would be to get a working developer environment going. Most of use use IntelliJ for that, but any setup supporting gradle should work. If you have questions feel free to reach out and ask me. I'd be happy to help

**Comment by panther2** [comment]
@kwaspek104 

Hey, for a start you also might want to look at

https://github.com/triplea-game/triplea/blob/master/docs/contribute.md#wrench-Developer-wo-Merge-Rights

Thank you 🙂
